subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -149,20 +149,15 @@ static int cmd_wifi_connect(const struct shell *shell, size_t argc,
 	struct net_if *iface = net_if_get_default();
 	static struct wifi_connect_req_params cnx_params;
 	char *endptr;
-	int idx = 3;
+	int idx = 2;
```

Comment:
Well, actually SSID is not a string, it's a bytearray. You could have anything inside (which was not parsed in shell though, so in a way it was half done, or half broken, depending on point of view)
However, all offloaded wifi API takes a string so I won't block this PR. But if one day zephyr has a Wifi soft MAC, SSID will have to be treated as byte array again.

Reply:
> I'll add data printing.

Thanks! As a packet can be big (1.5K for etherner), I guess it probably would make sense to print just beginning of it in this case, say, 100 bytes, clearly saying there're more (and print number of bytes like done now anyway too).

@tbursztyka, what do you think? ^^^